public/css/style.css

```diff
@@ -137,7 +145,6 @@ a {
   src: url("https://cdn.materialdesignicons.com/5.0.45/fonts/materialdesignicons-webfont.eot?v=5.0.45");
   src:
     url("https://cdn.materialdesignicons.com/5.0.45/fonts/materialdesignicons-webfont.woff2?v=5.0.45") format("woff2"),
-    url("https://cdn.materialdesignicons.com/5.0.45/fonts/materialdesignicons-webfont.woff?v=5.0.45") format("woff"),
```

Comment:
Do we care about IE 11- users this much or not?
If not, then we can get rid of the EOT URL and replace it with the WOFF2 one and possibly keep the WOFF as a fallback (although CanIUse shows an insignificant difference in support).

Reply:
@misterpeddy @AndyJiang99 @amirhmk

Reply:
I mean who really uses that...? And it's only font anyways right

Reply:
Yeah, speaking about MD web-fonts, I think that #60 will obliterate that especially since it seems to have lead to a **significant boost** in performance.